core/sail/lucene-api/src/main/java/org/eclipse/rdf4j/sail/lucene/config/AbstractLuceneSailConfig.java

```diff
@@ -85,14 +88,12 @@ public Set<String> getParameterNames() {
 	public Resource export(Model m) {
 		Resource implNode = super.export(m);
 
-		ValueFactory vf = SimpleValueFactory.getInstance();
-		m.setNamespace("sl", LuceneSailConfigSchema.NAMESPACE);
```

Comment:
do you want to add the config namespace to the model?

Reply:
Yeah the thinking was that this would be done at the top level already, but it doesn't hurt to be redundant.

Reply:
Done